cognee/api/v1/cognify/routers/get_cognify_router.py

```diff
@@ -6,22 +6,24 @@
 from cognee.modules.users.methods import get_authenticated_user
 from fastapi import Depends
 
+
 class CognifyPayloadDTO(BaseModel):
     datasets: List[str]
+    graph_model: BaseModel
```

Comment:
If we want to have graph_model be optional we should add the type in an Optional wrapper

Reply:
fixed